com.unity.cinemachine/Runtime/PostProcessing/FocusDistance.cs

```diff
@@ -19,21 +19,30 @@ static class Uniforms
         [Range(0, 1)]
         public float Stickiness = 0.4f;
 
+        [Tooltip("Radius of the FocusDistance sensor in the center of the screen.  A value of 1 would fill the screen.  "
+            + "It's recommended to keep this quite small.  Default value is 0.02")]
         [Range(0, 1)]
         public float KernelRadius = 0.02f;
 
+        [Tooltip("Depth tolerance for inclusion in the same depth bucket.  Effectively the depth resolution.")]
         [Range(0, 1)]
         public float DepthTolerance = 0.02f;
 
+        [Tooltip("Position on the screen of the depth sensor.  (0, 0) is screen center.")]
         public Vector2 ScreenPosition;
 
-        public ComputeShader m_ComputeShader;
-        public Camera m_Camera;
+        [Tooltip("Must be the FocusDistance compute shader.")]
+        public ComputeShader ComputeShader;
+
+        [Tooltip("The camera whose depth buffer will be checked.")]
+        public Camera Camera;
+
+        [Tooltip("If true, then the focus distance will be pushed to the camera's focusDistance field.")]
         public bool PushToCamera = true;
 
+        /// <summary>This holds the computed output.  Clients can read it as desired</summary>
         public float ComputedFocusDistance;
```

Comment:
Why not public for read only? and private for write?

Reply:
fixed